src/bundle/Resources/public/scss/ui/modules/universal-discovery/_menu.button.scss

```diff
@@ -1,7 +1,7 @@
 .c-menu-button {
     border: none;
     color: $ez-color-base-medium;
-    border-radius: calculateRem(2px);
+    border-radius: $border-radius;
```

Comment:
Do not use bootstrap variables, should be prefixed with `ibexa`, should be applied in more places